src/hb.hh

```diff
@@ -35,6 +35,10 @@
 #include "config.h"
 #endif
 
+#if defined (_MSC_VER) && defined (HB_BUILD_DLL)
```

Comment:
I wonder if we should check for DLL_EXPORT?  Is that standard name?

Reply:
DLL_EXPORT seems to me more of a MinGW thing, so I don't think it is exactly standard.